tests/unit/amo/reducers/test_errors.js

```diff
@@ -1,7 +1,7 @@
 import { createApiError } from 'amo/api/index';
 import { clearError, setError, setErrorMessage } from 'amo/actions/errors';
 import {
-  API_ERROR_SIGNATURE_EXPIRED,
+  API_ERRORS_SESSION_EXPIRY,
```

Comment:
We don't need to make these changes to this test either. It's just a coincidence that these tests are using `API_ERROR_SIGNATURE_EXPIRED` - they could have used any error. We are just checking behaviour that should be the case for any error, so there's no need to repeat the test with different codes.